stdlib/public/core/Unicode.swift

```diff
@@ -59,7 +59,9 @@ public protocol UnicodeCodecType {
   /// Because of buffering, it is impossible to find the corresponding position
   /// in the generator for a given returned `UnicodeScalar` or an error.
   ///
-  /// - parameter next: A generator of code units to be decoded.
+  /// - Parameter next: A generator of code units to be decoded.  Repeated
+  /// calls to this method on the same instance should always reuse the same
+  /// generator.  Failing to do so will result in undefined behavior.
```

Comment:
Please indent following lines of a bullet by two spaces.
What do you mean by “reuse” here?  Would “pass” be a better word?

I don’t think you mean undefined behavior; that actually includes memory and type-safety violations.  I think you mean the results are unspecified.  I think you also want to say that the generator or copies thereof shouldn’t be used for anything else between calls to this method.  This seems like a complicated-enough restriction to warrant an example of correct usage.